Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-drivers.md

```diff
@@ -35,6 +35,52 @@ example, if the Rook operator is running in the namespace `my-namespace` the
 provisioner value should be `my-namespace.rbd.csi.ceph.com`. The same provisioner
 name must be set in both the storageclass and snapshotclass.
 
+## Configure custom Driver name prefix for CSI Drivers
+
+To use a custom prefix for the CSI drivers, you need to set the
+`CSI_DRIVER_NAME_PREFIX` environment variable in the operator configmap. For
+instance, to use the prefix `my-prefix` for the CSI drivers, you can set the
```

Comment:
Don't say "you" in the documentation, just use the command form to be more direct.
```suggestion
instance, to use the prefix `my-prefix` for the CSI drivers, set the
```